codeship-steps.yml

```diff
@@ -6,5 +6,10 @@
   steps:
     #-  command: ./.ci/travis/travis.sh site
     #-  command: ./.ci/travis/travis.sh nondex
-    -  command: ls -la
-    -  command: .ci/run-link-check-plugin.sh
+    - command: ls -la
+    - command: SKIP_FILES=
+        ".github|appveyor.yml|circle.yml|distelli-manifest.yml|fast-forward-merge.sh|LICENSE|
```

Comment:
please use this form  https://github.com/checkstyle/checkstyle/blob/72a5294619a788a9e1c4e36e4df8177301ef1f9b/shippable.yml#L42

please make single `command` node with `- |` as it is done at https://github.com/checkstyle/checkstyle/blob/72a5294619a788a9e1c4e36e4df8177301ef1f9b/shippable.yml#L32

Reply:
Alright 👍

Reply:
Made the changes. Please reveiw